assets/src/components/lateView.tsx

```diff
@@ -261,4 +444,72 @@ const MissingLogonRow = ({
   )
 }
 
+const HideCheckbox = ({
+  vehicleOrGhost,
+}: {
+  vehicleOrGhost: VehicleOrGhost
+}): ReactElement<HTMLElement> | null => {
+  const { selectedIds, toggleCheckedState } = useContext(LateViewContext)
+  const isChecked = idIn(vehicleOrGhost.id, selectedIds)
+
+  return vehicleOrGhost.runId ? (
+    <input
+      type="checkbox"
+      defaultChecked={isChecked}
+      onClick={() => toggleCheckedState(vehicleOrGhost.runId!)}
+    />
+  ) : null
+}
+
+const HidePopup = ({
+  nRowsSelected,
+  hideSelectedRows,
+}: {
+  nRowsSelected: number
+  hideSelectedRows: () => void
+}) => (
+  <div>
+    {nRowsSelected} selected
+    <button onClick={hideSelectedRows}>Hide</button>
+  </div>
+)
+
+const UnhidePopup = ({
+  nRecentlyHidden,
+  unhideRecentlyHidden,
+  clearRecentlyHidden,
+}: {
+  nRecentlyHidden: number
+  unhideRecentlyHidden: () => void
+  clearRecentlyHidden: () => void
+}): ReactElement<HTMLElement> => {
+  const ref = useRef<HTMLDivElement>(null)
+
+  useEffect(() => {
```

Comment:
@lemald this is the start of the code I've been referring to that's giving me trouble.

Reply:
Hmm, no immediate theories come to mind but I will definitely keep thinking about it.

Reply:
@lemald cool, thanks for looking, this is puzzling because I directly stole this code from elsewhere in the project (`notificationDrawer.tsx` IIRC). Anyway, I proposed an alternative interaction to Tyler that should be more straightforward, and he said it would work, so change to come here.